### ISSUE TITLE
Server-side algo compatibility check

### ISSUE THREAD

**Issue by uz-spark** [Bendr0id/xmrigCC#329]
Check for whether an algo you push to the client is supported on it's version or not. Don't push if not supported, show error message.

**Comment by Bendr0id** [comment]
Good idea, but i currently dont know how this can be easily integrated.. The ".json" file is actually just processed as text and fille in the clientConfig for each miner and the server is not aware of version capabilities.

**Comment by uz-spark** [comment]
Store configs and templates in a memory data structure ?
